test/Desktop.js

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import requireUncached from 'require-uncached';
+
+describe('Desktop', () => {
+  describe('#get os', () => {
+    describe('process.platform', () => {
+      let _platform;
+      beforeEach(() => {
+        _platform = Object.getOwnPropertyDescriptor(process, 'platform');
+      });
+
+      afterEach(() => {
+        Object.defineProperty(process, 'platform', _platform);
+      });
+
+      it('works on win32', () => {
+        Object.defineProperty(process, 'platform', {
+          value: 'win32'
+        });
+        const Desktop = requireUncached('../src/Desktop');
+        expect(Desktop.os).to.equal('win');
+      });
+
+      it('works on darwin', () => {
+        Object.defineProperty(process, 'platform', {
+          value: 'darwin'
+        });
+        const Desktop = requireUncached('../src/Desktop');
+        expect(Desktop.os).to.equal('osx');
+      });
+
+      it('uses default:osx on other platform', () => {
+        Object.defineProperty(process, 'platform', {
+          value: 'other-os'
+        });
+        const Desktop = requireUncached('../src/Desktop');
+        expect(Desktop.os).to.equal('osx');
+      });
+    });
+
+    describe('navigator.userAgent', () => {
+      let _userAgent;
+      beforeEach(() => {
+        _userAgent = Object.getOwnPropertyDescriptor(navigator, 'userAgent');
+      });
+
+      afterEach(() => {
+        Object.defineProperty(navigator, 'userAgent', _userAgent);
+      });
+
+      it('works on Macintosh', () => {
```

Comment:
you seem to be testing for 'win' but the name of the test says 'works on Macintosh'